lms/templates/problem_ajax.html

```diff
@@ -1 +1 @@
-<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}"></div>
+<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}" data-content="${content | h}"></div>
```

Comment:
Looking closer at the CoffeeScript, I think @cpennington is right here: we should just render this right into the content block.

We can make it a flag, though -- `data-content-loaded`, or something -- that gets checked in lieu of content being passed into the `render` method.  We'd set that flag ourselves, and so we could call `render` to let it do everything else, but we wouldn't have the full contents of the block sitting out there on a data attribute.

It's also going to let the page render faster because we're not waiting on the JS code to execute and move it into place.

I'd say we should give it a shot.